CHANGES.md

```diff
@@ -1,6 +1,12 @@
 ﻿CKEditor 4 Changelog
 ====================
 
+## CKEditor 4.9.1
```

Comment:
Please fix version – as 4.9.1 is dropped, it should be 4.10.0

Reply:
I updated it to `4.10` to keep consistent with the rest of the file.